migrations/20160901_remove_null_notifications.js

```diff
@@ -0,0 +1,10 @@
+var _ = require('lodash');
+
+db.users.find({}).forEach(function(user){
```

Comment:
This migration file looks like it would work for a small database, but I bet it times out running across a database of over a million users. 

Let's just remove this file for now and one of the DB admins will write the migration file.